samples/esb/esp_ptx/README.rst

```diff
@@ -79,19 +72,19 @@ After programming the Transmitter sample on one of the development kits and the
 Dependencies
 ************
 
-This sample uses the following |NCS| libraries:
+This sample uses the following |NCS| library:
 
 * :ref:`esb_readme`
 
 In addition, it uses the following Zephyr libraries:
 
-* :file:`include/zephyr/types.h`
+* ``include/zephyr/types.h``
```

Comment:
```suggestion
* :file:`include/zephyr/types.h`
```
Use this tagging for all files.

Reply:
Is this what is causing the doc build error perhaps?

Reply:
Don't think so. The doc build error messages from CI are more or less useless nowadays.